scripts/build/build.ps1

```diff
@@ -15,6 +15,18 @@ $ErrorActionPreference="Stop"
 
 . "$RepoRoot\scripts\build\generate-version.ps1"
 
+if ($env:CI_BUILD -eq "1") {
```

Comment:
Because in the Unix build we set `$HOME` to `artifacts\home` so it's already being done.